src/components/ManageLoginData.jsx

```diff
@@ -0,0 +1,19 @@
+import React,{useState} from "react";
+import Login from "./Login";
+
+function ManageLoginData () {
+    const [login,setLogin] = useState({
+        username : "",
+        password : "",
+    });
+
+    function onLoginChange (event){
+        const updateDetails = {...login, [event.target.name]: event.target.value};
+        setLogin(updateDetails);
+    }
```

Comment:
```suggestion
    const onLoginChange = (newState) => {
        setLogin({...login, ...newState});
    }
```
Where the `newState` value is something like `{ username: 'exampleusername' }`.

Reply:
using it this way does not set the key so when trying to input something it will not allow please advice

Reply:
So then change the way you are handling the onChange function of the input to produce this.